src/server/common/dbconn.cpp

```diff
@@ -109,6 +110,61 @@ std::string dbconn::get_tournaments()
 	}
 }
 
+std::unique_ptr<simple_wml::document> dbconn::get_game_history(int player_id, int offset)
+{
+	try
+	{
+		std::string game_history_query = "select "
+"  game.GAME_NAME, "
+"  game.RELOAD, "
+"  game.START_TIME, "
+"  GROUP_CONCAT(CONCAT(player.USER_NAME, ':', player.FACTION)) as PLAYERS, "
+"  IFNULL(scenario.NAME, scenario.ID) as SCENARIO, "
```

Comment:
Probably better to provide the client with all info and let it decide what to display.

For example the client might want to display "not set" in the user's language and the ID in a tooltip or so.

Reply:
Alright, will do.